doc/rados/operations/erasure-code.rst

```diff
@@ -115,6 +115,20 @@ no two *chunks* are stored in the same rack.
 More information can be found in the `erasure code profiles
 <../erasure-code-profile>`_ documentation.
 
+Erasure code profiles overhead calculation
+------------------------------------------
```

Comment:
https://docs.ceph.com/docs/master/rados/operations/erasure-code/#erasure-code-profiles section talks a bit about overhead, how about we use the same example and move this there?